spec/installer-spec.js

```diff
@@ -37,7 +41,4 @@ test('creates a nuget package and installer', async t => {
   if (process.platform === 'win32') {
     t.true((await stat(path.join(outputDirectory, 'MyAppSetup.msi'))).isFile())
   }
-
-  log('Verifying Update.exe');
-  t.true((await stat(path.join(appDirectory, 'Update.exe'))).isFile());
```

Comment:
The Update.exe file get embedded into the nuget package and therefore into the installer. The file will not be present in the appDirectory or the outputDirectory either.

A different unit test can be written to verify the content of the nuget package file.